bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/events/ItemStateChangedEvent.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.items.events;
+
+import org.eclipse.smarthome.core.events.AbstractEvent;
+import org.eclipse.smarthome.core.types.State;
+
+/**
+ * {@link ItemStateChangedEvent}s can be used to deliver item state changes through the Eclipse SmartHome event bus. In
+ * contrast to the {@link ItemStateEvent} the {@link ItemStateChangedEvent} is only sent if the state changed.
+ * State events must be created with the {@link ItemEventFactory}.
+ *
+ * @author Dennis Nobel - Initial contribution
+ */
+public class ItemStateChangedEvent extends AbstractEvent {
+
+    /**
+     * The item state changed event type.
+     */
+    public final static String TYPE = ItemStateChangedEvent.class.getSimpleName();
+
+    private final String itemName;
+
+    private final State itemState;
+
+    /**
+     * Constructs a new item state changed event.
+     *
+     * @param topic the topic
+     * @param payload the payload
+     * @param itemName the item name
+     * @param newItemState the new item state
+     */
+    protected ItemStateChangedEvent(String topic, String payload, String itemName, State newItemState) {
```

Comment:
If we can assume that the event publisher always knows the old state (which should be the case here, since he otherwise wouldn't know that it has changed), I think this is a good idea for state changes (i.e. also for ThingStatusChanged events). Especially as the current rule engine also has triggers like "changed from x to y". Just FTR: For other events (especially with a more complex payload like Notifications and Things), I am not a big fan of sending the "old" version, since it duplicates the data and is usually tough to interpret anyhow.